src/test/css/css-parser_test.ts

```diff
@@ -26,11 +26,11 @@ suite('CssParser', () => {
 
     setup(() => {
       parser = new CssParser(<any>{
-        findImports(url: string, document: CssDocument): Array<any>/* */ {
-            return [];  //
+        findImports: function(url: string, document: CssDocument): Array<any> {
```

Comment:
It's really unfortunate to not use arrow functions and methods in object literals. what's going on here?